core/rawdb/ancient_scheme.go

```diff
@@ -58,13 +63,13 @@ const (
 	stateHistoryStorageData  = "storage.data"
 )
 
-// stateFreezerNoSnappy configures whether compression is disabled for the state freezer.
-var stateFreezerNoSnappy = map[string]bool{
-	stateHistoryMeta:         true,
-	stateHistoryAccountIndex: false,
-	stateHistoryStorageIndex: false,
-	stateHistoryAccountData:  false,
-	stateHistoryStorageData:  false,
+// stateFreezerTableConfigs configures whether compression is disabled for the state freezer.
+var stateFreezerTableConfigs = map[string]freezerTableConfig{
+	stateHistoryMeta:         {snappy: true, prunable: false},
```

Comment:
It's wrong. All the tables in state freezer should be prunable.

Reply:
`prunable` is only for `TruncateTailBlocks`, which doesn't really work on state history I think.